src/icons/states/SecurityBlockedState.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray10, bdlGray50, white } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    strokeColor?: string,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const SecurityBlockedState = ({
+    className = '',
+    color = bdlGray10,
+    height = 167,
+    strokeColor = bdlGray50,
+    title,
+    width = 130,
+}: Props) => (
+    <AccessibleSVG
+        className={`bdl-SecurityBlockedState ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 130 167"
+        width={width}
+    >
+        <path
+            className="stroke-color"
+            stroke={color}
```

Comment:
It seems a little confusing that some `stroke-color` classed elements are using `color` vs `strokeColor`.

Reply:
Changed it to primary and secondary since there are two colors in this one